### ISSUE TITLE
Returns all data as zero when timeframe set to today 5-y 

### ISSUE THREAD

**Issue by Reynolddoss** [GeneralMills/pytrends#592]
Data gets retrieved but the values of all weeks seem to be zero.

```
from pytrends.request import TrendReq
pytrends = TrendReq(hl='en-US', tz=360)
pytrends.build_payload(["Real Madrid CF"], cat=0, timeframe='today 5-y', geo='', gprop='')
df = pytrends.interest_over_time()
```
![image](https://github.com/GeneralMills/pytrends/assets/26821831/f4d288ae-b664-40a3-865b-e4870cdbd257)

**Comment by ShannonWavelet** [comment]
Also having the same issue with a lot of consecutive zero values in the dataset downloaded

**Comment by nicktba** [comment]
This is a known issue. 

The problem is when the Google Trends API is requested it's flagged as USER_TYPE_SCRAPER and returns data with 0s. 

The issue is on Google's side, it's even affecting the embedding feature as you can see. This is because the embedding feature also makes a request to the Google Trends API.

<img width="621" alt="Screenshot 2023-07-20 at 11 14 30 AM" src="https://github.com/GeneralMills/pytrends/assets/85900436/eefc7009-91aa-4874-a535-32f990dceb75">

All other SERP products are facing this issue and a solution is not known. 

When this happened last time, it persisted for a few weeks and then resolved itself. So that's what we can hope for.

**Comment by Raidus** [comment]
@nicktba tested out the embeddings approach too. its often not even loading for me. 

![image](https://github.com/GeneralMills/pytrends/assets/8803882/e6258d06-6b48-4793-aaa9-9c53caadbcce)

https://8pf86r.csb.app/

**Comment by nicktba** [comment]
Yep, check your network headers, you'll see its a 429 error blocking the request. 

You'll also see `USER_TYPE_SCRAPER` In those headers. 

That's because nearly all requests to the Google Trends API is being blocked, including the native embedding feature. 

That's how we know its an internal issue and there is little we can do as of now.

**Comment by Raidus** [comment]
Looks like it's been resolved by Google. I get again normal data points and no errors. Also Iframes work again.

Update: Experiencing the issue again

**Comment by nicktba** [comment]
> Looks like it's been resolved by Google. I get again normal data points and no errors. Also Iframes work again.
> 
> Update: Experiencing the issue again

It seems like I'm getting somewhat better success, but the issue not fixed by any means
